### PR TITLE
[BH-1550] Medstats translations

### PR DESCRIPTION
**Pull request by swichu91**
Updated meditation statistics translations.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: image/assets/lang/Polski.json [hunk 1]
```diff
@@ -727,5 +727,9 @@
   "app_bell_meditation_chime_interval_bottom": "medytacji",
   "app_bell_meditation_chime_volume": "Głośność dzwonka interwału",
   "app_meditation_summary": "<text>Medytowałeś przez<br />",
-  "app_meditation_countdown_desc": "Startuje za"
+  "app_meditation_countdown_desc": "Startuje za",
+  "app_meditation_summary_total": "<text>Suma :<br /><token>$VALUE</token></text>",
```
Comment (patrycja-paczkowska) [inline image/assets/lang/Polski.json:731]: unnecessary space

File: image/assets/lang/Espanol.json [hunk 2]
```diff
@@ -708,5 +708,8 @@
   "app_bell_meditation_chime_interval_bottom": "de la meditación",
   "app_bell_meditation_chime_volume": "Volumen del timbre",
   "app_meditation_summary": "<text>Has meditado por<br />",
-  "app_meditation_countdown_desc": "Empieza en"
+  "app_meditation_countdown_desc": "Empieza en",
+  "app_meditation_summary_total": "<text>Total:<br /><token>$VALUE</token></text>",
+  "app_meditation_summary_average": "Promedio/por día:",
+  "app_meditation_summary_title": "<text> últimos <token>$VALUE</token> días</text>"
```
Comment (patrycja-paczkowska) [inline image/assets/lang/Espanol.json:714]: should start with a capital letter

File: image/assets/lang/Francais.json [hunk 3]
```diff
@@ -678,5 +678,8 @@
   "app_bell_meditation_chime_interval_bottom": "de la méditation",
   "app_bell_meditation_chime_volume": "Volume du gong intervalle",
   "app_meditation_summary": "<text>Vous avez médité pendant<br />",
-  "app_meditation_countdown_desc": "Début dans"
+  "app_meditation_countdown_desc": "Début dans",
+  "app_meditation_summary_total": "<text>Total:<br /><token>$VALUE</token></text>",
```
Comment (sofianemudita) [inline image/assets/lang/Francais.json:682]: In French colons should be proceeded by space -> Total :

File: image/assets/lang/Francais.json [hunk 4]
```diff
@@ -678,5 +678,8 @@
...
+  "app_meditation_summary_average": "Moyenne/jour:",
```
Comment (sofianemudita) [inline image/assets/lang/Francais.json:683]: In French colons should be proceeded by space -> Moyenne/jour :

File: image/assets/lang/Espanol.json [hunk 5]
```diff
@@ -709,5 +709,8 @@
   "app_bell_meditation_chime_interval_bottom": "de la meditación",
   "app_bell_meditation_chime_volume": "Volumen del timbre",
   "app_meditation_summary": "<text>Has meditado por<br />",
-  "app_meditation_countdown_desc": "Empieza en"
+  "app_meditation_countdown_desc": "Empieza en",
+  "app_meditation_summary_total": "<text>Total:<br /><token>$VALUE</token></text>",
+  "app_meditation_summary_average": "Promedio/por día:",
+  "app_meditation_summary_title": "<text> Últimos <token>$VALUE</token> días</text>"
```
Comment (patrycja-paczkowska) [inline image/assets/lang/Espanol.json:715]: no space at the beginning

**Review by patrycja-paczkowska — APPROVED** [review]

**Pull request merged**